<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarDetail</class>
 <widget class="QDialog" name="CarDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车辆详细信息</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>671</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>车辆信息</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>41</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0,0">
     <item row="0" column="3">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="labelCondition">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="labelBrand">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="labelPerform">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>车牌号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>车型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>型号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelKeeper">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>车况</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>状态</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>购买日期</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>持有者</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="labelDateofBuy">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="tableViewInsure">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>671</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewMaintain">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>671</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>保险信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>保养信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>760</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
